### DISCUSSION TITLE
Multiplying two exp_op operators

### DISCUSSION THREAD

**Discussion by rishkhare** [QuSpin/QuSpin#424] [Q&A] [answered]
I was wondering whether it is possible to multiply two exp_op operators. When I use the dot function, it gives me an error since it expects one of them to be a vector.

A workaround is to use get_mat for the objects and then just use numpy.matmul() but I fear that it will slow the code for larger systems.

Can anyone help me with this?

**Comment by mgbukov** [comment, marked as answer]
I think using `get_mat` should not incur any extra computational costs; the reason is that `get_mat` distinguishes between sparse and dense matrices before computing the matrix exponential. 

In the future, we can consider adding the option to multiply two `exp_op` objects but this will most likely be a trivial wrapper.
